src/test/cdk/explorer/constructNode.test.ts

```diff
@@ -46,6 +40,15 @@ describe('ConstructNode', function () {
         assert.strictEqual(resourceUri.fragment, testNode.resource.construct.path)
     })
 
+    it('initializes icon paths for CDK constructs', async function () {
+        const testNode = new ConstructNode(location, {
+            ...treeUtils.generateConstructTreeEntity('', constructTreePath),
+            attributes: treeUtils.generateAttributes(),
+        })
+
+        assert.strictEqual(testNode.treeItem.iconPath, getIcon('aws-cloudformation-stack'))
```

Comment:
TODO: this test passes against an outdated CDK tree format but not against a new one
So CFN stack icons do not show up, not sure when this change occurred.